src/http/ngx_http_mruby_module.c

```diff
@@ -23,6 +23,10 @@
 #include <mruby/value.h>
 #include <mruby/version.h>
 
+#if (NGX_HTTP_SSL)
+#include <openssl/ssl.h>
+#endif
+
```

Comment:
it is to solve `SSL_verify_cb` symbol.
well, It can be removed by defining `typedef int SSL_verify_cb(int ok, X509_CTX *)`.

I would remove some if pragma in `ngx_mruby_module.h` for it.

Reply:
ngx_core.h already included openssl/ssl.h. So, you don't include the header file, I guess.